src/crypto/index.js

```diff
@@ -609,12 +612,14 @@ Crypto.prototype.bootstrapSecretStorage = async function({
                         };
                     }
 
+                    // use the backup key as the new ssss key
+                    ssssKeys[newKeyId] = backupKey;
+                    opts.key = backupKey;
```

Comment:
Do we need to make a similar change near "Secret storage default key not found" below (line 648) which also adds a 4S key?

Reply:
This will be handled by the change at https://github.com/matrix-org/matrix-js-sdk/pull/1294/files#diff-fa4472db829c5a59596c29a2f43e8f9cR390 because react-sdk uses that function to get the backup key.